skins/cat17/templates/partials/donation/donor_form.html.twig

```diff
@@ -1,28 +1,13 @@
 {% set COUNTRIES = [ 'DE', 'AT', 'CH', 'BE', 'IT', 'LI', 'LU' ] %}
-{% set PAYMENT_INTERVALS = [ 0, 1, 3, 6, 12 ] %}
 
-<div class="introduction container default">
-	<div class="row">
```

Comment:
Is this now intentionally missing from the main form?

Reply:
The actual donation form is located at skins/cat17/templates/Donation_Form.html.twig. This file is only for the confirmation page, I think I will rename it to make that more clear.

Reply:
Ah, I was under the assumption that you're re-using the `donor_form.html.twig` partial on `Donation_Form.html.twig` and that removing the text would break `Donation_Form.html.twig`.

Since I still think re-using the form fields is a good idea, I suggest the following: 

* Split `donor_form` into `partials/donor_update/donor_form.html.twig` and `partials/donation/donor_fields.html.twig`, the latter only has the container elements with the fields
* Replace donor field markup in `Donation_Form.html.twig` with an include of `partials/donation/donor_fields.html.twig`.

Reply:
Hey, I don't think that will work because technically the forms differ, the donor update form does not include the anonymous donor selector and I think your solution would mean that both forms share the exact same structure. Though maybe I am misunderstanding you? Or is that what you meant and I should just hide the anonymous selector with CSS for the donor update use case?

Reply:
Ah ok, in this case just move the partial to a different directory.

While I'm not totally against hiding stuff with CSS, I prefer not to generate the markup if it does not complicate the code otherwise.